### PR TITLE
Fix broken master

### PR DESCRIPTION
**Pull request by wetted**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -6,6 +6,13 @@ managed-mailjet = "5.2.1"
 
 poi-ooxml = "5.2.3"
 kotlin = "1.7.21"
+spock = "2.3-groovy-4.0"
+testcontainers = "1.17.6"
+
+micronaut-aws = "3.9.2"
+micronaut-reactor = "3.0.0-SNAPSHOT"
+micronaut-serde = "2.0.0-SNAPSHOT"
+micronaut-views = "3.7.1"
```
Comment (timyates) [inline gradle/libs.versions.toml:15]: Can you raise an issue to fix this once we get a snapshot of views?

Also AWS

**Review by timyates — APPROVED** [review]
Now we wait for the actions 😁

File: gradle/libs.versions.toml (hunk 1 above)
Comment (wetted) [reply gradle/libs.versions.toml:15]: #169

**Comment by wetted** [comment]
@timyates if we can merge this, I can also take care of #167 which has breaking changes that require code and catalog refactor

**Pull request merged**